jujugui/static/gui/src/app/store/env/bakery.js

```diff
@@ -75,14 +80,8 @@ YUI.add('juju-env-bakery', function(Y) {
         } else {
           this.visitMethod = this._defaultVisitMethod;
         }
-        // If there is a cookie that may already be set, such as when coming
-        // from the storefront, use that cookie name instead of the generated
-        // one.
-        if (cfg.existingCookie) {
-          this.macaroonName = cfg.existingCookie;
```

Comment:
I'm confused why we are now throwing away the existing cooking name? Wouldn't we want to keep that?